ui/ducks/send/send.js

```diff
@@ -1821,14 +1844,18 @@ export function updateSendAsset({ type, details }) {
  * passing in both the dispatch method and the payload to dispatch, which makes
  * it only applicable for use within action creators.
  */
-const debouncedValidateRecipientUserInput = debounce((dispatch, payload) => {
-  dispatch(
-    addHistoryEntry(
-      `sendFlow - user typed ${payload.userInput} into recipient input field`,
-    ),
-  );
-  dispatch(validateRecipientUserInput(payload));
-}, 300);
+const debouncedValidateRecipientUserInput = debounce(
```

Comment:
The addition of resolve as a parameter allows the caller to asynchronously await the debounced call